dummy-api/matrix-service-api.yaml

```diff
@@ -0,0 +1,115 @@
+swagger: '2.0'
```

Comment:
Do we want to specify _BasePath_ - perhaps /v0 ?

Reply:
Yeah. The azure doc you've linked suggests there are a few schemes to accomplish this, but basePath is the one I'm most familiar with.